src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java

```diff
@@ -64,435 +64,435 @@ public class MinestuckEnUsLanguageProvider extends MinestuckLanguageProvider
-		add(MSBlocks.STAT_STORER, "Stat Storer");
-		addTooltip(MSBlocks.STAT_STORER, "Lets off a redstone signal with the strength of the signal based on the active value type stored.");
-		add(MSBlocks.REMOTE_OBSERVER, "Remote Observer");
-		addTooltip(MSBlocks.REMOTE_OBSERVER, "Lets off a full strength redstone signal if the active condition is met.");
-		add(MSBlocks.WIRELESS_REDSTONE_TRANSMITTER, "Wireless Redstone Transmitter");
-		addTooltip(MSBlocks.WIRELESS_REDSTONE_TRANSMITTER, "Sends a signal to Wireless Redstone Receiver at the input coordinates if the coordinates are loaded.");
-		add(MSBlocks.WIRELESS_REDSTONE_RECEIVER, "Wireless Redstone Receiver");
-		addTooltip(MSBlocks.WIRELESS_REDSTONE_RECEIVER, "Receives signals from Wireless Redstone Transmitters.");
-		add(MSBlocks.SOLID_SWITCH, "Solid Switch");
-		addTooltip(MSBlocks.SOLID_SWITCH, "KACHUNK");
-		add(MSBlocks.VARIABLE_SOLID_SWITCH, "Variable Solid Switch");
-		addTooltip(MSBlocks.VARIABLE_SOLID_SWITCH, "The functionality of a solid switch but throw in the ability to change what redstone power it gives off.");
-		add(MSBlocks.ONE_SECOND_INTERVAL_TIMED_SOLID_SWITCH, "Timed Solid Switch (One Second)");
-		addTooltip(MSBlocks.ONE_SECOND_INTERVAL_TIMED_SOLID_SWITCH, "Loses one value of power every second until it turns off.");
-		add(MSBlocks.TWO_SECOND_INTERVAL_TIMED_SOLID_SWITCH, "Timed Solid Switch (Two Second)");
-		addTooltip(MSBlocks.TWO_SECOND_INTERVAL_TIMED_SOLID_SWITCH, "Loses one value of power every two seconds until it turns off.");
-		add(MSBlocks.SUMMONER, "Summoner");
-		add(MSBlocks.AREA_EFFECT_BLOCK, "Area Effect Block");
-		addExtra(MSBlocks.AREA_EFFECT_BLOCK, AreaEffectBlock.EFFECT_CHANGE_MESSAGE, "Effect type changed to %s with the amplification strength %s.");
-		add(MSBlocks.PLATFORM_GENERATOR, "Platform Generator");
-		add(MSBlocks.PLATFORM_BLOCK, "Platform Block");
-		add(MSBlocks.PLATFORM_RECEPTACLE, "Platform Receptacle");
-		add(MSBlocks.ITEM_MAGNET, "Item Magnet");
-		add(MSBlocks.REDSTONE_CLOCK, "Redstone Clock");
-		add(MSBlocks.ROTATOR, "Rotator");
-		add(MSBlocks.TOGGLER, "Toggler");
-		add(MSBlocks.REMOTE_COMPARATOR, "Remote Comparator");
-		add(MSBlocks.STRUCTURE_CORE, "Structure Core");
-		add(MSBlocks.FALL_PAD, "Fall Pad");
-		add(MSBlocks.FRAGILE_STONE, "Fragile Stone");
-		add(MSBlocks.RETRACTABLE_SPIKES, "Retractable Spikes");
-		add(MSBlocks.BLOCK_PRESSURE_PLATE, "Block Pressure Plate");
-		add(MSBlocks.PUSHABLE_BLOCK, "Pushable Block");
-		add(MSBlocks.AND_GATE_BLOCK, "AND Gate");
-		add(MSBlocks.OR_GATE_BLOCK, "OR Gate");
-		add(MSBlocks.XOR_GATE_BLOCK, "XOR Gate");
-		add(MSBlocks.NAND_GATE_BLOCK, "NAND Gate");
-		add(MSBlocks.NOR_GATE_BLOCK, "NOR Gate");
-		add(MSBlocks.XNOR_GATE_BLOCK, "XNOR Gate");
-		add(MSBlocks.GATE, "Gate");
-		add(MSBlocks.RETURN_NODE, "Return Node");
+		add(MSBlocks.BLACK_CHESS_DIRT.get(), "Black Chess Tile");
```

Comment:
Use `addBlock()` instead.
Example:
`addBlock(MSBlocks.BLACK_CHESS_DIRT, "Black Chass Tile");`

Reply:
Now have done so. Also changed addTooltip to addBlockTooltip, addExtra to addBlockExtra, etc